docs/articles/Getting-Started.md

```diff
@@ -0,0 +1,386 @@
+and view our Holocron module in the browser. In another terminal window you can run
+
+```bash
+npm run watch:build
+```
+
+and this will watch for any changes made to your module, then update `one-app-runner`
+with the rebuilt module bundle. The command above uses `one-app-bundler` which can
+also be used to bundle our Holocron module.
+
+```bash
+npm run build
+```
+
+When ready to publish to production, make sure to set `NODE_ENV=production` before building.
+
+**Local Configuration**
+
+Inside the `package.json` of your Holocron module, you can
+include a `"one-amex": {}` property to configure some of the
+tools in our ecosystem.
+
+```json
+{
+  "name": "my-module",
+  "one-amex": {
+    "bundler": {
+      "webpackConfigPath": "webpack.config.js"
+    },
+    "runner": {
+      "rootModuleName": "my-module",
+      "modules": [
+        ".",
+        "../my-adjacent-child-module"
+      ]
+    }
+  }
+}
+```
+
+When you start using child modules, `one-app-runner` can include
+multiple local modules when it's configured to accept them.
+`one-app-bundler` can have it's `webpack` config extended along
+with other options.
+
+> #### More Info
+>
+> **Recipes**
+>
+> [Run One App Locally](../recipes/running-one-app-locally)
+>
+> [Running In Production](../recipes/running-in-production)
+>
+> [Publishing Modules](../recipes/publishing-modules)
+>
+> ##### Packages
+>
+> [`@americanexpress/one-app-runner`](https://github.com/americanexpress/one-app-cli/tree/main/packages/one-app-runner)
+>
+> [`@americanexpress/one-app-bundler`](https://github.com/americanexpress/one-app-cli/tree/main/packages/one-app-bundler)
+
+### Adding CSS Styles
+
+Styling a Holocron module with `CSS` or `SCSS` can be accomplished by creating a
+separate file that is used with the markup we write in our module.
+
+`src/components/styles.scss`
+
+```css
+.myButton {
+  background-color: green;
+
+  &:active,
+  &:focus,
+  &:hover {
+    background-color: blue;
+  }
+}
+```
+
+We can import the stylesheet into our module and use CSS modules to access the class name
+for each selector by its name. The class names are unique when they are generated which
+avoids unwanted cascading of styles in our document.
+
+`src/components/MyModule.jsx`
+
+```jsx
+import React from 'react';
+import styles from './styles.scss';
+
+export default function MyModule() {
+  return (
+    <div>
+      <button type="button" className={styles.myButton}>
+        Click Me
+      </button>
+    </div>
+  );
+}
+```
+
+> #### More Info
+>
+> **Recipes**
+>
+> [Adding Styles Recipe](../recipes/adding-styles)
+
+### Creating Routes
+
+One App has built in dynamic routing that uses each Holocron module to
+build out the router. There is a special property that we can assign to
+our module `Module.childRoutes` which would allow us to add routes to
+One App. `childRoutes` should be assigned a function that is given
```

Comment:
It only needs to be a function when you need to use the store for something, otherwise it can be an array or just a route with or without nested routes

Reply:
TIL, thank you for sharing that 😄